Python/pywarpx/_libwarpx.py

```diff
@@ -1210,7 +1210,7 @@ def get_mesh_phi_fp(level, include_ghosts=True):
     return _get_mesh_field_list(libwarpx.warpx_getPhiFP, level, None, include_ghosts)
 
 
-def get_gathered_phi_fp(level):
+def get_gathered_phi_fp(level, include_ghosts=True):
```

Comment:
We should remove this entire wrapper as well as the `get_gathered_rho_fp` wrapper. We can call `_get_mesh_field_list(libwarpx.warpx_getGatheredChargeDensityFP, level, None, include_ghosts)` directly from mwxrun.py.